shared/actions/signup/index.js

```diff
@@ -306,9 +306,9 @@ function signup (skipMail: boolean, onDisplayPaperKey?: () => void): TypedAsyncA
             onDisplayPaperKey && onDisplayPaperKey()
             dispatch(nextPhase())
           },
-          'keybase.1.gpgUi.wantToAddGPGKey': (params, {error, result}) => {
```

Comment:
@keybase/react-hackers this is a subtle thing i want to point out. the response object is actually a class instance so this is dangerous behavior. if you call the naked result() there is no this pointer!